core/controllers/pages.py

```diff
@@ -82,6 +82,15 @@ def get(self):
         self.render_template('pages/about/about.html')
 
 
+class StaticAbout(base.BaseHandler):
```

Comment:
Why do we need this handler at all?

Can we just add a clause in app.yaml to map the URL to the static page directly?

Reply:
There are still some jinja tags link `{% macro warnings_and_loader() -%}`.
Also, [render_template](https://github.com/oppia/oppia/blob/develop/core/controllers/base.py#L302) adds some variables and meta info to the page.

I think it can be statically served once I get rid of these.